examples/cfd/09_Darcy_flow_equation.ipynb

```diff
@@ -0,0 +1,436 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    class GaussianRF(object):
</pre><p>Any reason those use <code>torch</code> instead of <code>numpy</code>.I understand that your application may be ML related but this is a larger dependencies so I would stick to <code>numpy</code> for this tutorial that implements all these methods as well.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1998/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>